src/ecdsa/__init__.py

```diff
@@ -44,7 +44,6 @@
     "numbertheory",
     "test_pyecdsa",
     "util",
-    "six",
```

Comment:
yes, it's wrong but it's here for backwards compatibility

Reply:
ok, I'll readd it